linera-execution/tests/test_execution.rs

```diff
@@ -1313,7 +1393,19 @@ async fn test_multiple_messages_from_different_applications() -> anyhow::Result<
     // Execute the operation, starting the test scenario
     let context = create_dummy_operation_context();
     let mut controller = ResourceController::default();
-    let mut txn_tracker = TransactionTracker::new(0, Some(Vec::new()));
+    let mut txn_tracker = TransactionTracker::new(
+        0,
+        0,
+        Some(
+            caller_blobs
+                .iter()
+                .chain(silent_blobs.iter())
+                .chain(sending_blobs.iter())
+                .copied()
+                .map(OracleResponse::Blob)
+                .collect(),
```

Comment:
This is repeated a lot. Maybe some helper method that turns an iterator of blobs into a vector of oracle responses would be nice.